lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -1534,13 +3400,13 @@
                 dateTimeFormat.__templateString = templateString;
 
                 /*
-                 * NOTE:
-                 * Pattern string's are position-sensitive; while templates are not.
-                 * If we specify {hour.integer(2)}:{minute.integer(2)} pattern string; we will always format as HH:mm.
-                 * On the other hand, template strings don't give as fine granularity for options; and the platform decides how long month.abbreviated should be.
-                 * Therefore, we have to create using template strings; and then change the .abbreivated/.integer values to have correct digits count if necessary.
-                 * Thus, this results in this redundant looking code to create dateTimeFormat twice.
-                 */
+                    * NOTE:
+                    * Pattern string's are position-sensitive; while templates are not.
+                    * If we specify {hour.integer(2)}:{minute.integer(2)} pattern string; we will always format as HH:mm.
+                    * On the other hand, template strings don't give as fine granularity for options; and the platform decides how long month.abbreviated should be.
+                    * Therefore, we have to create using template strings; and then change the .abbreivated/.integer values to have correct digits count if necessary.
+                    * Thus, this results in this redundant looking code to create dateTimeFormat twice.
+                    */
```

Comment:
This was introduced by a `Format Selection` that I ran after copying master's Intl code here, itll be gone soon enough so I am not worried about it.